webapp/components/Form.tsx

```
@@ -397,6 +396,46 @@ const ReceiptForm = (): JSX.Element => {
+                        type="submit"
+                        color={
+                          submitting || hasValidationErrors
+                            ? 'default'
+                            : 'success'
+                        }
+                        className={
+                          'mt-4 ' +
+                          (submitting || hasValidationErrors
+                            ? 'cursor-not-allowed'
+                            : 'cursor-pointer')
+                        }
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)